app/src/main/java/org/wikipedia/page/PageActivity.kt

```diff
@@ -391,6 +392,13 @@ class PageActivity : BaseActivity(), PageFragment.Callback, LinkPreviewDialog.Lo
         return super.onKeyDown(keyCode, event)
     }
 
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        if (resultCode == RESULT_OK && data?.hasExtra(OnThisDayGameFinalFragment.EXTRA_GAME_COMPLETED) == true) {
+            OnThisDayGameFinalFragment.maybeShowOnThisDayGameEndMessage(this)
+        }
+    }
```

Comment:
I am not able to see the snackbar. Could you please share the steps to make it show up?